amivapi/settings.py

```diff
@@ -97,30 +96,19 @@
 CONFIRM_TEXT = "Your singup was confirmed!"
 SIGNUP_DELETED_TEXT = "Your signup was removed."
 
-# All organisational units (ou) in ldap which are assigned to AMIV (by VSETH)
-LDAP_MEMBER_OU_LIST = [
-    u'Biomedical Engineering MSc',
-    u'Dr. Informationstechnologie und Elektrotechnik',
-    u'Dr. Maschinenbau und Verfahrenstechnik',
-    u'DZ Elektrotechnik und Informationstechnologie',
-    u'DZ Maschineningenieurwiss. und Verfahrenstechnik',
-    u'Elektrotech. und Informationstechnol. (Mobilität)',
-    u'Elektrotechnik und Informationstechnologie BSc',
-    u'Elektrotechnik und Informationstechnologie MSc',
-    u'Energy Science and Technology MSc',
-    u'Informationstechnologie und Elektrotechnik',
-    u'Maschinenbau und Verfahrenstechnik',
-    u'Maschineningenieurwissenschaften (Mobilität)',
-    u'Maschineningenieurwissenschaften BSc',
-    u'Maschineningenieurwissenschaften MSc',
-    u'Micro- and Nanosystems MSc',
-    u'Nuclear Engineering MSc',
-    u'Nuclear Engineering MSc (EPFL)',
-    u'Robotics, Systems and Control MSc',
-    u'Verfahrenstechnik MSc',
-    u'Doktorat Informationstechnologie & Elektrotechnik',
-    u'Doktorat Maschinenbau und Verfahrenstechnik'
-]
+
+# In LDAP, the 'departmentNumber' field contains the students departments
+# It contains: type of person, e.g. student, department, exact field of study
+# We can use this to discover department of students
+LDAP_DEPARTMENT_MAP = {
```

Comment:
Is this also tested with mobility students?

Reply:
Yes, it also works with mobility students.